content/locales/en.ts

```diff
@@ -41,22 +41,100 @@ export const en = {
     title: "Personal information",
     breadcrumb: "Profile | new",
     info: "Please fill in these information to create your profile.",
+    namePlaceholder: "Enter your full name",
+    usernamePlaceholder: "Enter your username",
+    profilePlaceholder: "Enter your profile URL",
+    websitePlaceholder: "Enter your website URL",
+    picturePlaceholder: "Enter your picture URL",
+    zoneInfoPlaceholder: "Enter your zone info",
+    localePlaceholder: "Enter your language",
+    nameLabel: "Full name *",
+    usernameLabel: "Username",
+    birthDateLabel: "Birth date",
+    profileLabel: "Profile URL",
+    websiteLabel: "Website URL",
+    pictureLabel: "Picture URL",
+    zoneInfoLabel: "Zone info",
+    localeLabel: "Language",
+    add: "Add my information",
+    update: "Update my information",
+    cancel: "Cancel",
```

Comment:
Maybe we could make this kind of duplicate things separate objects?
So when we would have to change them, we won't have to change it twice?
Wdyt?

Reply:
I thought about it. Have them in a separate object and spread it when needed. I'll check if there are no issues with it. But yeah I think that's the way we should do it!